plugins/plugin-java-debugger/che-plugin-java-debugger-server/src/main/java/org/eclipse/che/plugin/jdb/server/JavaDebugger.java

```diff
@@ -115,7 +115,7 @@
     /** Current thread. Not <code>null</code> is thread suspended, e.g breakpoint reached. */
     private ThreadReference thread;
     /** Current stack frame. Not <code>null</code> is thread suspended, e.g breakpoint reached. */
-    private JdiStackFrame   stackFrame;
+    private JdbStackFrame   stackFrame;
```

Comment:
JDI it's a "java debugger interface". If you rename this field, I think you should rename  getJdiStackFrame method too.